---
title: Investigating genotype-phenotype relationships for tuber bruising in autotetraploid potatoes
subtitle: Keynote speaker
excerpt: Presentation of the research I've done during my PhD.
author: Olivia Angelin-Bonnet, Susan Thomson, Samantha Baldwin, Patrick J. Biggs and Matthieu Vignes
event: MAPNET 2022
event_url: https://www.scienceevents.co.nz/mapnet
date: "2022-03-15"
location: Online
categories:
- genetics
- conference
draft: false
featured: true
layout: single
show_post_time: false

links:
- icon: door-open
  icon_pack: fas
  name: Conference website
  url: https://www.scienceevents.co.nz/mapnet
---

## Abstrat

Tuber bruising of tetraploid potato is an important quality trait as it affects the appearance and
flavour of the tubers and thus impacts their fitness for sale. The development of potato lines that
are more resistant to bruising is therefore a desirable objective for breeding programs,
rendering the genetic analysis of this trait an important task. In this study, we investigated the
biological mechanisms underlying tetraploid potato tuber bruising using multi-omics data.
Genotype by sequencing using exon capture obtained from a breeding population of halfsibling families was used to uncover regions of interest for the bruising phenotype, as well as
other agronomic traits of interest. In addition, we employed a Systems Biology approach to
obtain a more holistic and comprehensive view of the molecular mechanisms involved in tuber
bruising, and bridge the gap between genetic variations and phenotype. To this end, RNA
sequencing and metabolomics data were obtained, and GWAS, differential expression analysis
and multi-omics data integration were leveraged in order to detect molecular features (i.e.
genomic variants, genes and metabolites) involved in tuber bruising. We demonstrate that even
as capture sequencing only allows us to measure genetic variations in a subset of the genome,
it is possible to uncover interesting and biologically meaningful genotype phenotype
associations, especially when combining the GWAS results with other omics datasets.
Moreover, these associations were obtained with samples selected from a breeding program,
demonstrating that available data from populations not specifically designed for association
study can be used to uncover genomic regions potentially associated with a trait of interest.
